Flaky test `NodeArrayFutureTest.testTimeoutIsSpread()`

The following failure sometimes happen with JDK 17:

```
[ERROR] Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 79.05 s <<< FAILURE! -- in org.mortbay.jetty.orchestrator.NodeArrayFutureTest
[ERROR] org.mortbay.jetty.orchestrator.NodeArrayFutureTest.testTimeoutIsSpread -- Time elapsed: 14.65 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Expected java.util.concurrent.TimeoutException to be thrown, but nothing was thrown.
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:152)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:73)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3115)
	at org.mortbay.jetty.orchestrator.NodeArrayFutureTest.testTimeoutIsSpread(NodeArrayFutureTest.java:135)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
```

--- a/src/test/java/org/mortbay/jetty/orchestrator/NodeArrayFutureTest.java
+++ b/src/test/java/org/mortbay/jetty/orchestrator/NodeArrayFutureTest.java
@@ -131,8 +131,12 @@ public class NodeArrayFutureTest extends AbstractSshTest
         try (Cluster cluster = new Cluster(cfg))
         {
             NodeArray nodeArray = cluster.nodeArray("my-array");
-            NodeArrayFuture future = nodeArray.executeOnAll(tools -> Thread.sleep(600));
-            assertThrows(TimeoutException.class, () -> future.get(1, TimeUnit.SECONDS));
+            NodeArrayFuture future = nodeArray.executeOnAll(tools ->
+            {
+                int id = tools.barrier("testTimeoutIsSpread-barrier", 2).await();
+                Thread.sleep(1600L * (id + 1L));
+            });
+            assertThrows(TimeoutException.class, () -> future.get(3, TimeUnit.SECONDS));
         }
     }
 }

